src/pudl/metadata/classes.py

```diff
@@ -526,6 +527,22 @@ def to_rst(
         )
         return rendered
 
+    def generate_encodable_data(self: Self, size: int = 10) -> pd.Series:
```

Comment:
I created this for testing, and it was useful, so I thought I would leave it.  Maybe there's  a relevant unit test that should be created for the coding process?

Reply:
I created a unit test that makes use of this function to generate encodable data, and then encode it for each encoder.

Reply:
that sounds very handy overall! and it seems like this code should live over in the unit test.

Reply:
I could turn it into a function that takes an Encoder in the test modules, but unfortunately then it can't be imported for use in development or to mess around in a notebook.